README.md

```diff
@@ -4,7 +4,7 @@
 
 # [Cloud Logging for Bunyan: Node.js Client](https://github.com/googleapis/nodejs-logging-bunyan)
 
-[![release level](https://img.shields.io/badge/release%20level-general%20availability%20%28GA%29-brightgreen.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
```

Comment:
The content of the `README.md` is autogenerated. This change would be reverted by the owlbot.
What is a reason for this change? We use the same shields in all client libraries.

Reply:
This change to the README is generated automatically by owlbot from .repo-metadata.json. This was not a manual change. The templates have been updated so this change should be reverted soon.